expertise/create_dataset.py

```diff
@@ -46,22 +46,30 @@ def get_profile_ids(openreview_client, group_ids):
     :param group_ids: List of group ids
     :type group_ids: list[str]
 
-    :return: List of tilde ids
+    :return: List of tuples containing (tilde_id, email)
     :rtype: list
     """
-    tilde_members = []
+    members = []
     for group_id in group_ids:
         group = openreview_client.get_group(group_id)
+
         profile_members = [member for member in group.members if '~' in member]
+        profile_search_results = openreview_client.search_profiles(emails=None, ids=profile_members, term=None)
+        tilde_members = []
+        for profile in profile_search_results:
+            preferredEmail = profile.content.get('preferredEmail') or profile.content.get('emails')[0]
+            tilde_members.append((profile.id, preferredEmail))
+        members.extend(tilde_members)
+
         email_members = [member for member in group.members if '@' in member]
+        email_set = set(email_members)
```

Comment:
@carlosmondra You are not using this `email_set` anywhere. So, let's remove it. Since these are group members, they will always be unique anyway.